types/lodash/lodash-tests.ts

```diff
@@ -4291,6 +4291,18 @@ fp.now(); // $ExpectType number
     _(42).isObject(); // $ExpectType boolean
     _.chain([]).isObject(); // $ExpectType LoDashExplicitWrapper<boolean>
     fp.isObject(anything); // $ExpectType boolean
+    if (fp.isObject(anything)) {
+        anything; // $ExpectType object
+    }
+    if (_.isObject(anything)) {
+        anything; // $ExpectType object
+    }
+    if (_(anything).isObject()) {
+        anything; // $ExpectType any
+    }
+    if (_.chain(anything).isObject()) {
+        anything; // $ExpectType any
+    }
```

Comment:
I couldn't find a way to use user-defined type guards for the implicit and explicit wrappers.

Reply:
For implicit wrappers, it kinda works:
```ts
    interface LoDashImplicitWrapper<TValue> {
        isObject(): this is LoDashImplicitWrapper<object>;
    }

    const wrapper = _(anything);
    if (wrapper.isObject()) {
        wrapper.value(); // $ExpectType object
    }
```

But not for explicit ones. Anyways, you forgot to call `.value()` in their test code:
```ts
    if (_.chain(anything).isObject().value()) {
        anything; // $ExpectType any
    }
```

Reply:
Fixed, thanks

Reply:
It seems unnecessary to check that `anything` *still* has type `any` - I think it's best to just remove these last 2 tests.